packages/ses/src/error/tame-console.js

```diff
@@ -76,17 +114,42 @@ export const tameConsole = (
     globalThis.window !== undefined &&
     globalThis.window.addEventListener !== undefined
   ) {
-    // eslint-disable-next-line @endo/no-polymorphic-call
     globalThis.window.addEventListener('error', event => {
-      // eslint-disable-next-line @endo/no-polymorphic-call
       event.preventDefault();
-      // eslint-disable-next-line @endo/no-polymorphic-call
       causalConsole.error(event.error);
       if (errorTrapping === 'exit' || errorTrapping === 'abort') {
         globalThis.window.location.href = `about:blank`;
       }
     });
+
+    // Maybe track unhandled promise rejections.
+    if (unhandledRejectionPeriodMs && globalThis.WeakRef !== undefined) {
+      const scheduleOnce = callback => {
+        globalThis.setTimeout(callback, unhandledRejectionPeriodMs);
```

Comment:
Ok, I just do not understand why timeout is the right tool for this.

Reply:
My mistake.  The timer was dead code that I just didn't realize could be entirely removed.